Meta.Tests.ps1

```diff
@@ -1,57 +1,52 @@
 <# 
-.summary
-    Test that describes code.
-#>
+    .summary
+        Test that describes code.
 
+    .PARAMETER Force
+        Used to force any installations to occur without confirming with
+        the user.
+#>
 [CmdletBinding()]
-param()
+Param (
+    [Boolean]$Force = $false
+)
 
 if (!$PSScriptRoot) # $PSScriptRoot is not defined in 2.0
 {
     $PSScriptRoot = [System.IO.Path]::GetDirectoryName($MyInvocation.MyCommand.Path)
 }
+# Make sure MetaFixers.psm1 is loaded - it contains Get-TextFilesList
+Import-Module -Name (Join-Path -Path $PSScriptRoot -ChildPath 'MetaFixers.psm1') -Force
+
+# Load the TestHelper module which contains the *-ResourceDesigner functions
+Import-Module -Name (Join-Path -Path $PSScriptRoot -ChildPath 'TestHelper.psm1') -Force
 
 $ErrorActionPreference = 'stop'
 Set-StrictMode -Version latest
 
 $RepoRoot = (Resolve-Path $PSScriptRoot\..).Path
 
-Import-Module $PSScriptRoot\MetaFixers.psm1
-
-#region Download and import latest stable DSC Resource Designer module
-
-# We are using nuget to be compatible with WMF v4 machines which
-# do not have the PowerShell Package Management module.
-$DesignerModuleName = 'xDscResourceDesigner'
-$DesignerModulePath = "$env:USERPROFILE\Documents\WindowsPowerShell\Modules\$DesignerModuleName"
-$nugetSource = 'https://www.powershellgallery.com/api/v2'
-$nugetOutputDirectory = "$(Split-Path -Path $DesignerModulePath -Parent)\\"
-
-
-if (Test-Path -Path $DesignerModulePath)
-{
-    # Remove any installed version of the DscResourceDesigner module
-    Remove-Item -Path $DesignerModulePath -Recurse -Force
+# Install and/or Import xDSCResourceDesigner Module
+if ($env:APPVEYOR) {
+    # Running in AppVeyor so force silent install of xDSCResourceDesigner
+    $PSBoundParameters.Force = $true
```

Comment:
The check for $env:AppVeyor isn't strictly required, but if that code is removed all existing AppVeyor.yml configs would need to have the following line added:
```
Install-ResourceDesigner -force
```
Added. E.g.:
```
#---------------------------------#
#      environment configuration  #
#---------------------------------#
os: Unstable
version: 2.4.{build}.0
install:
  - cinst -y pester
  - git clone https://github.com/PowerShell/DscResource.Tests
  - ps: Push-Location
  - ps: Import-Module DscResource.Tests\TestHelper.psm1 -force
  - ps: Install-ResourceDesigner -force
.......
```

Which I think would be a real headache (and any ones that didn't get update will start failing). This code ensures that no matter what happens existing projects will continue to work as normal.